aiutils/vis/image_io.py

```diff
@@ -1,54 +1,55 @@
-from termcolor import colored
 from PIL import Image
 import numpy as np
 import pdb
 
-
-def raise_exception(e, err_str):
-    pkg_err_str = 'Exception raised in package pyAIUtil: \n{}'.format(err_str)
-    print colored(pkg_err_str, 'red')
-    raise
-
-
-def check_assertion(cond, err_str):
-    pkg_err_str = '\nAssertion failed in package pyAIUtil: \n{}'.format(
-        err_str)
-    assert (cond), colored(pkg_err_str, 'blue')
+import aiutils.devtools.err_msg as err_msg
 
 
 def imread(filename):
+    """
+    Matlab like function for reading an image file.
+
+    Returns:
+    np_im (numpy.ndarray): h x w x 3 ndarray for color images and 
+        h x w for grayscale images
+    """
     try:
         im = Image.open(filename)
-
     except IOError as e:
```

Comment:
It produces colored messages which I thought should help in identifying issues in the package vs errors thrown by some other library. I don't know if this would help or create more nuisance. But it seems the more exceptions we handle with useful error messages, we would be in for less of a surprise when we actually start using the package and start running into bugs.